src/api/users/src/services/firestore.js

```diff
@@ -1,23 +1,41 @@
 const admin = require('firebase-admin');
 const { logger } = require('@senecacdot/satellite');
+const fs = require('fs');
+const path = require('path');
 
-const {
-  FIREBASE_TYPE,
-  FIREBASE_PROJECT_ID,
-  FIREBASE_PRIVATE_KEY_ID,
-  FIREBASE_PRIVATE_KEY,
-  FIREBASE_CLIENT_EMAIL,
-  FIREBASE_CLIENT_ID,
-  FIREBASE_AUTH_URI,
-  FIREBASE_TOKEN_URI,
-  FIREBASE_AUTH_PROVIDER_X509_CERT_URL,
-  FIREBASE_CLIENT_X509_CERT_URL,
-} = process.env;
+let FIREBASE_TYPE;
+let FIREBASE_PROJECT_ID;
+let FIREBASE_PRIVATE_KEY_ID;
+let FIREBASE_PRIVATE_KEY;
+let FIREBASE_CLIENT_EMAIL;
+let FIREBASE_CLIENT_ID;
+let FIREBASE_AUTH_URI;
+let FIREBASE_TOKEN_URI;
+let FIREBASE_AUTH_PROVIDER_X509_CERT_URL;
+let FIREBASE_CLIENT_X509_CERT_URL;
+
+try {
+  const privateKey = fs.readFileSync(path.join('/app', 'serviceAccountKey.json'));
```

Comment:
In node.js, you can `const values = require('./file.json')` and it will load and parse it in one step

Reply:
What is `/app`?  This needs a comment above it explaining the assumptions you're making here, which seem to be many.

Reply:
I added a comment but to clarify:  `/app` is the folder where the keys will live on prod (as suggested by Josue when him and I were discussing this) 😄